samples/dashboards/edit-mode/ctx-enabled/demo.css

```diff
@@ -0,0 +1,3 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/css/datagrid.css");
```

Comment:
The `datagrid.css` is unnecessary, because of the missing datagrid component in the config.